uint64 utime
string ctrl_fault_status
uint8 mech_id
uint8 joint_id
uint8 num_mechanisms
string[] mech_fault_status

